packages/app/src/update/constants/environment.constant.ts

```diff
@@ -3,8 +3,8 @@
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
 
-export enum ReleaseSpace {
-  ProductionReleases = "latest",
-  PreReleases = "rc-latest",
-  DailyReleases = "daily-latest",
+export enum Environment {
```

Comment:
I'm wondering if to mark explicitly that this Environment is about OS, not the Center app. What do you think? 🤔

Reply:
Good idea, definitely makes sense. Will change that

Reply:
Changed in https://github.com/mudita/mudita-center/pull/1057/commits/1a969057b644e80909f6a8bc8e3d66c4bc287e8f

(not only `Environment`, but some other enums/interfaces)